package.json

```diff
@@ -14,12 +15,14 @@
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "format": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "test": "echo \"TODO: add tests\" && exit 0"
   },
   "dependencies": {
     "@ardrive/turbo-sdk": "^1.1.0",
```

Comment:
can you test v1.1.1-alpha-1 and see if it removes the need for vite polyfill plugin below?

Reply:
https://github.com/ardriveapp/turbo-app/pull/7

Reply:
all good - will continue validating on that branch